src/net/native-stack.cc

```diff
@@ -427,6 +439,17 @@ std::vector<network_interface> native_network_stack::network_interfaces() {
     return res;
 }
 
+} // namespace net
+
+namespace internal {
+
+void increment_bytes_sent(uint64_t bytes) {
+    net::bytes_sent += bytes;
 }
+void increment_bytes_received(uint64_t bytes) {
+    net::bytes_received += bytes;
+}
```

Comment:
This must be in the same translation unit as its used in, statistics must compile to a single ADD instruction.